packages/realm/src/app-services/MutableSubscriptionSet.ts

```diff
@@ -104,7 +104,7 @@ export class MutableSubscriptionSet extends BaseSubscriptionSet {
    * @param query A {@link Results} instance representing the query to remove a subscription to.
    * @returns `true` if the subscription was removed, `false` if it was not found.
    */
-  remove(query: Results<unknown>): boolean {
+  remove(query: Results<any>): boolean {
```

Comment:
Why change this to `any`?

Reply:
It should have probably been a part of a separate PR.

This is the error I was seeing in the flexible sync integration tests:

![Screenshot 2023-02-13 at 12 49 08](https://user-images.githubusercontent.com/1243959/218450512-142c1208-c8db-4e30-96be-7884d53c78cc.png)

Reply:
We have quite a few TS issues in the flexible sync tests, so perhaps we should add a task to go over those and figure out the root cause of this in the process?

Reply:
Yeah, let's make an issue for it 👍🏼